stingray/varenergyspectrum.py

```diff
@@ -265,12 +364,16 @@ def _spectrum_function(self):
         pass
 
 
-class RmsEnergySpectrum(VarEnergySpectrum):
+class RmsSpectrum(VarEnergySpectrum):
```

Comment:
Why did you change the name?

Reply:
Could there not be an RMSFrequencySpectrum? In which case, this would be confusing

Reply:
The RMSFrequencySpectrum is just a Powerspectrum in `frac` or `abs` normalization :)

Reply:
I just thought that, since we are in the varenergyspectrum module, all the classes are energy spectra, and it was redundant. I left the old names in those classes that already existed, for compatibility (but I don't think many people actually used this machinery).